packages/core/src/servient.ts

```diff
@@ -160,6 +160,21 @@ export default class Servient {
         }
     }
 
+    public destroyThing(thingId: string): Promise<boolean> {
+        return new Promise<boolean>((resolve, reject) => {
+            if (this.things.has(thingId)) {
+                console.debug("[core/servient]", `Servient destroying thing with id '${thingId}'`);
+                this.things.delete(thingId);
+                let serverPromises: Promise<boolean>[] = [];
+                this.servers.forEach((server) => { serverPromises.push(server.destroy(thingId)); });
```

Comment:
What I found a little "concerning" is that we ourselves are not using the returned boolean value... it seems like we introduced it but we don't need it.

Reply:
I see what you are heading at but I am not sure whether it might turn out useful in the future.

I still think it does not hurt and we can go on but if you feel uncomfortable I am also OK with changing...

Reply:
Let's go on, for now, the core <-> protocol binding interface it's still evolving.